workers/unity/Packages/com.improbable.gdk.core/Systems/CleanReactiveComponentsSystem.cs

```diff
@@ -11,26 +13,26 @@ namespace Improbable.Gdk.Core
     [UpdateInGroup(typeof(SpatialOSSendGroup.InternalSpatialOSCleanGroup))]
     public class CleanReactiveComponentsSystem : ComponentSystem
     {
-        private MutableView view;
-
         private readonly List<Action> removeComponentActions = new List<Action>();
 
         // Here to prevent adding an action for the same type multiple times
         private readonly HashSet<Type> typesToRemove = new HashSet<Type>();
 
+        public readonly Dictionary<int, ComponentTranslation> TranslationUnits =
```

Comment:
This isn't used at all I think? The translation units are all fetched from the worker?

Reply:
yep, forgot to remove this after changing other stuff